CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## 1.0.3 (2020-09-29)
```

Comment:
Doesn't necessarily need a release of its own, could sit in dev till we have something else to release. But otherwise LGTM